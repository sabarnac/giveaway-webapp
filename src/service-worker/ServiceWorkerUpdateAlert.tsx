import { inject } from "mobx-react";
import { withTranslation } from "react-i18next";
import ServiceWorkerUpdateAlert from "./ServiceWorkerUpdateAlert.react";
import { createObserver } from "../util";

/**
 * React component for the service worker add alert.
 */
export default withTranslation()(
  inject("serviceWorkerAlertsConfig")(createObserver(ServiceWorkerUpdateAlert)),
);
