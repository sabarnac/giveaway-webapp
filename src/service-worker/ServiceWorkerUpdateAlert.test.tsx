import React from "react";
import ServiceWorkerUpdateAlert from "./ServiceWorkerUpdateAlert";
import { create, ReactTestRenderer } from "react-test-renderer";

describe("ServiceWorkerUpdateAlert Component.", () => {
  it("Matches snapshot.", () => {
    const component: ReactTestRenderer = create(<ServiceWorkerUpdateAlert />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
