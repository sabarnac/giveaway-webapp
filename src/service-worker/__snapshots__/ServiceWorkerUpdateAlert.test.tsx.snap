// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceWorkerUpdateAlert Component. Matches snapshot. 1`] = `
<div
  onConfirm={[Function]}
  show={true}
  text="serviceWorker.updateMessage.message"
  title="serviceWorker.updateMessage.title"
  type="warning"
/>
`;
